bash.html.markdown

```diff
@@ -25,7 +25,21 @@ Nearly all examples below can be a part of a shell script or executed directly i
 # As you already figured, comments start with #. Shebang is also a comment.
 
 # Simple hello world example:
-echo Hello world!
+echo "Hello world!"
+
+# Color output of echo command
+# This command will show green 'Hello world!' text
+echo -e "\033[0;32mHello world!\033[0m"
+
+# You can use these codes
+Black        0;30     Dark Gray     1;30
```

Comment:
This command will show color output of echo command. You can use different codes for colors 0;31 -red 0;30 black  etc.